repos/system_upgrade/el7toel8/actors/cupsfilterscheck/actor.py

```diff
@@ -0,0 +1,49 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import Report, InstalledRedHatSignedRPM, CupsFiltersModel
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+from subprocess import Popen, PIPE
+
+class CupsfiltersCheck(Actor):
+    """
+    cupsfilters_check actor
+
+    Checks if cups-filters package is installed and if cups-browsed.conf was
+    modified by user, which prevents automatic replacement by package installer.
+    """
+
+    name = 'cupsfilters_check'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report, CupsFiltersModel)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+      installed_cupsfilters = False
+      migrateable = False
+
+      for rpm_pkgs in self.consume(InstalledRedHatSignedRPM):
+        for pkg in rpm_pkgs.items:
+          if pkg.name == "cups-filters":
+            installed_cupsfilters = True
+
+      if installed_cupsfilters is False:
+        report_generic(
+                        title='cups-filters will not be migrated',
+                        summary='cups-filters is not installed',
+                        severity='low'
+                      )
```

Comment:
There is nothing wrong codewise with the call to report_generic. But, reports will be read by sys admins/developers after the upgrade process. I, personally, think we should keep it as concisely as possible to avoid readers skipping it and missing valuable information. Since there is no action that user should take if cups-filters is not installed, what do you think we change this for a debug message or even remove it at all?

Reply:
Please remove this report, as it is not useful to anyone. If you want, feel free to log it via `self.log.info`